docs/overview.md

```diff
@@ -882,4 +882,37 @@ Dendrograms are tree-like driagrams that represent hierarchical relationships an
 ![](images/complete-linkage-full.png)
 
 
-In our context, the height of the dendrogram represents the **distance between clusters** or the **cophenetic distance** calculated during the hierarchical clustering process. The cophenetic distance of two objects (samples or hierarchical clusters) is a measure of how similar those objects need to be in order to be grouped into the same cluster. This distance is the height of the dendrogram at which point two objects are hierarchically clustered together. For example, when interpreting the above dendrogram, we see that the cluster `(C,(A,B))` has a dendrogram height of `4`. This means that the cophenetic distance between clusters `(C)` and `(A,B)` is `4`. The specific interpretation of the distance varies depending on the linkage method used: average, single, or complete. Regardless, the greater the height of a hierarchical cluster in a dendrogram, the more dissimilarity there is between the clusters that comprise it.
+In our context, the height of the dendrogram represents the **distance between clusters** or the **cophenetic distance** calculated during the hierarchical clustering process. The cophenetic distance of two objects (samples or hierarchical clusters) is a measure of how similar those objects need to be in order to be grouped into the same cluster. This distance is the height of the dendrogram at which point two objects are hierarchically clustered together (in contrast to the **patristic distance** or sum of branch lengths between two objects, see [Patristic and Cophenetic definitions][]). For example, when interpreting the above dendrogram, we see that the cluster `(C,(A,B))` has a dendrogram height of `4`. This means that the cophenetic distance between clusters `(C)` and `(A,B)` is `4`. The specific interpretation of the distance varies depending on the linkage method used: average, single, or complete. Regardless, the greater the height of a hierarchical cluster in a dendrogram, the more dissimilarity there is between the clusters that comprise it.
```

Comment:
"(in contrast to the **patristic distance** or sum of branch lengths between two objects, see [Patristic and Cophenetic definitions][])"

We should be careful about this, because the main reason that patristic and cophenetic distances are difference from each other (in this context) is because of how we're defining the translation of the dendrogram to a tree. UPGMA is _one_ way of translating this, but it's not the only way, which means that patristic and cophenetic are not intrinsically different in all contexts.

Reply:
The reason why I didn't explicitly say something like this when writing this explanation is because in order to make a statement that they're different, you need to explain why they're different (mainly UPGMA distributes the clustering/dendrogram heights in a specific way // being ultrametric). We could add such an explanation, but it'll require being very confident about our understanding.

Reply:
Thanks. I tried to re-word and fix up our definitions to give a bit more context. Please let me know what you think. eb6f533a3cd07c65220d3208a4a3fdc236ea6554